src/components/Trade/TradeInput.tsx

```diff
@@ -94,6 +109,14 @@ export const TradeInput = () => {
   const { getAvailableSwappers } = getSwappersApi.endpoints
   const { tradeQuoteArgs } = useTradeQuoteService()
   const dispatch = useAppDispatch()
+  const [isManualAddressEntryValidating, setIsManualAddressEntryValidating] = useState(false)
+  const isYatFeatureEnabled = useFeatureFlag('Yat')
+
+  const {
+    formState: { errors: manualAddressEntryErrors },
+    trigger: manualAddressEntryValidationTrigger,
+    setValue: setManualReceiveAddressValue,
```

Comment:
Not sure about the terminology here, this seems form-specific and not related to the manual receive address featureset?

Reply:
Yeah that's fair - addressed in https://github.com/shapeshift/web/pull/4479/commits/ae7253c23f0352f585bab637223ea053ef5e3843, though kept the "form" vernacular because there is so bloody much going on in this file.